en/pe-docs/docs/devops/automation-pipelines/run-automation-pipeline.md

```
@@ -0,0 +1,56 @@
+
+## Manage Existing Pipeline Variables and Secrets
+
+To update or remove a secret, click the three vertical dots in the upper-right corner of the variable box in the Choreo UI. From the menu, choose the appropriate action.
+
+## Pipeline Parameters
+
+Parameters are another type of variables referred by automation pipelines. Speciality of these variables is, before running the pipeline, user needs to choose a value for the parameters out of specified.
+
+In pipeline definition, specify parameters like below.
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)